cloudtrust-common/pom.xml

```diff
@@ -15,6 +15,13 @@
     <properties>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
+
+        <version.pmd.plugin>3.13.0</version.pmd.plugin>
+        <jacoco.version>0.8.5</jacoco.version>
+        <spotbugs.version>4.0.0</spotbugs.version>
+        <fb-contrib.version>7.4.7</fb-contrib.version>
```

Comment:
Move these properties to cloudtrust-parent so that they can be used in all child module

Reply:
Good suggestion. Initially I did not want to touch the parent pom. I've changed the PR for that in a new commit.